libfio.c

```diff
@@ -353,14 +353,17 @@ int initialize_fio(char *envp[])
 	 * can run into problems on archs that fault on unaligned fp
 	 * access (ARM).
 	 */
+	compiletime_assert((offsetof(struct thread_data, ts) % 8) == 0, "ts");
```

Comment:
Should that be sizeof a pointer instead of 8 here? The others are all explicit 64-bit types, hence the 8.